static/docs/commands-reference/import-url.md

```diff
@@ -345,63 +298,71 @@ $ tree
 ```
 
-DVC has noticed the external dependency (import stage) has changed. It is
-telling us that it is necessary to now run `dvc repro`.
+DVC has noticed the "external" source (import stage) has changed! This tells us
+that it is necessary to now run `dvc repro` so that the rest of this pipeline
+(i.e. the "prepare" stage) is automatically run again.
+
+> Note that had we not unlocked the import stage before, we could alternatively
+> run `dvc update` on it first to make sure it's up to date, and then check with
+> `dvc status` for the need to reproduce the pipeline.
+
+Lets run the "prepare" stage again, since we know its outdated:
 
 ```dvc
 $ dvc repro prepare.dvc
-
-WARNING: Dependency '/path/to/data-store/data.xml' of 'data.xml.dvc' changed because it is 'modified'.
+WARNING: Dependency '../../../tmp/dvc-import-url-example/data.xml' of 'data.xml.dvc' changed because it is 'modified'.
 WARNING: Stage 'data.xml.dvc' changed.
 Reproducing 'data.xml.dvc'
-Importing '/path/to/data-store/data.xml' -> 'data/data.xml'
+Importing '../../../tmp/dvc-import-url-example/data.xml' -> 'data/data.xml'
 [##############################] 100% data.xml
-Saving 'data/data.xml' to cache '.dvc/cache'.
+[##############################] 100% data.xml
+
+Saving 'data/data.xml' to '.dvc/cache/bf/23db315e2d9b25d3f30038c49a812e'.
+
 Saving information to 'data.xml.dvc'.
 
 WARNING: Dependency 'data/data.xml' of 'prepare.dvc' changed because it is 'modified'.
-WARNING: Stage 'prepare.dvc' changed.
-Reproducing 'prepare.dvc'
-Running command:
-    python src/prepare.py data/data.xml
-Saving 'data/prepared' to cache '.dvc/cache'.
-Linking directory 'data/prepared'.
-Saving information to 'prepare.dvc'.
 
-To track the changes with git run:
+WARNING: Stage 'prepare.dvc' changed.
 
-    git add data.xml.dvc prepare.dvc
+Reproducing 'prepare.dvc'
 
-$ git add .
-$ git commit -a -m "updated data"
+Running command:
+	python src/prepare.py data/data.xml
 
-[master a8d4ce8] updated data
- 2 files changed, 6 insertions(+), 6 deletions(-)
+Output 'data/prepared' didn't change. Skipping saving.
 
+Saving information to 'prepare.dvc'.
+...
 $ dvc status
 Pipelines are up to date. Nothing to reproduce.
 ```
 
-Because the external data source for the data file changed, the change was
-noticed by the `dvc status` command. Running `dvc repro` then ran both stages of
-this pipeline, and if we had set up the other stages they also would have been
-run. It first downloaded the updated data file. And then noticing that
-`data/data.xml` had changed, that triggered the `prepare.dvc` stage to execute.
+`dvc repro` ran again both stages of this pipeline (and if we had set up more
+stages, they also would have been run again). It first downloaded the updated
+data file. And then, noticing that `data/data.xml` had changed, that triggered
+the `prepare.dvc` stage to execute.
+
+> Feel free to clean up after this example get out of and remove the
```

Comment:
I'm not sure it's important

Reply:
Me neither. I guess not! Removed in 1a214ea as well.